agora-scripts/ScriptExport/Options.hs

```diff
@@ -1,38 +1,32 @@
 {- |
-Module     : Options
+Module     : ScriptExport.Options
 Maintainer : emi@haskell.fyi
 Description: Command line options for 'agora-scripts'.
 
 Command line options for 'agora-scripts'.
 -}
-module Options (Options (..), parseOptions) where
+module ScriptExport.Options (Options (..), parseOptions) where
 
+import Network.Wai.Handler.Warp qualified as Warp
 import Options.Applicative ((<**>))
 import Options.Applicative qualified as Opt
 
-data Options = Options
-  { config :: FilePath
-  , output :: FilePath
+newtype Options = Options
+  { port :: Warp.Port
   }
   deriving stock (Show, Eq)
 
 opt :: Opt.Parser Options
 opt =
   Options
-    <$> Opt.strOption
-      ( Opt.long "config"
-          <> Opt.short 'c'
-          <> Opt.metavar "CONFIG_PATH"
-          <> Opt.value "./agora-scripts/agora-params.json"
+    <$> Opt.option
+      Opt.auto
+      ( Opt.long "port"
+          <> Opt.short 'p'
+          <> Opt.metavar "PORT"
+          <> Opt.value 3939
           <> Opt.help "The path where the script configuration is."
```

Comment:
This help message is outdated.

Reply:
And also, I think we need a toggle for CORS workaround? I imagine this server will run behind a reverse proxy when we actually deploy it.

Reply:
Yep, will change both of these.